### ISSUE TITLE
[enhancement]: AzureStaticWebApp@0 - select docker image

### ISSUE THREAD

**Issue by peruzzof** [microsoft/azure-pipelines-tasks#18255]
### Task name

AzureStaticWebApp@0

### Describe your feature request here

Since 04/05 I am facing an issue with the AzureStaicWebApp task execution, as I am already speaking with the Oryx mantainer (https://github.com/microsoft/Oryx/issues/1981), I would like to be able to select the label of the docker image being used. In fact seems that if I run with "prev" I will have a workaround for this issue.
So, in summary, I would like a new flag to select the version of the staticappsclient being used, with the allowed values "prev", "latest" and "stable" (default)
